frontend/src/router.js

```diff
@@ -447,6 +519,12 @@ export function periodRoute(period, routeName = 'camp/period/program', query = {
   }
 }
 
+export function materialFromRoute(route) {
```

Comment:
```suggestion
export function materialListFromRoute(route) {
```
and i would move this method to the other ...FromRoute methods

Reply:
[fb0e02c](https://github.com/ecamp/ecamp3/pull/3797/commits/fb0e02c95cafbb7cc387d2ce5fbbd32af23c3f8c)